platform/android/java/build.gradle

```diff
@@ -36,8 +36,11 @@ allprojects {
 
 ext {
     supportedAbis = ["armv7", "arm64v8", "x86", "x86_64"]
-    supportedTargetsMap = [release: "release", dev: "debug", debug: "release_debug"]
     supportedFlavors = ["editor", "template"]
+    supportedTargetsMapByFlavors = [
+        "editor": [release: "release_debug", dev: "debug", debug: "release_debug"],
```

Comment:
Change `release` builtype for the editor to `storeRelease`

Reply:
On further investigations, I'm sticking with `release` because every Android project has an automatic `release` build type. Changing it to `storeRelease` will require additional logic to disable the default `release` build type so we may as well leverage what already exists.